api/lib/infrastructure/repositories/complementary-certification-course-result-repository.js

```diff
@@ -18,6 +18,18 @@ module.exports = {
     );
   },
 
+  async getPixSourceResultByComplementaryCertificationCourseId({ complementaryCertificationCourseId }) {
+    const result = await knex
+      .select('complementary-certification-course-results.*')
```

Comment:
histoire d'en rajouter:
```suggestion
      .select('*')
```

Reply:
Done.